app/models/accounting/quickbooks/transaction_creator.rb

```diff
@@ -10,9 +10,10 @@ def initialize(root_division = Division.root)
         @principal_account = root_division.principal_account
       end
 
-      def add_disbursement(amount:, loan_id:, memo:, description:, qb_bank_account_id:, qb_customer_id:)
+      def add_disbursement(amount:, loan_id:, memo:, description:, qb_bank_account_id:, qb_customer_id:, date: nil)
         je = ::Quickbooks::Model::JournalEntry.new
         je.private_note = memo
+        je.txn_date = date if date.present?
```

Comment:
Most of this PR is beefing up the specs, but this is the code that was requested in the ticket.